docs/examples/example_feature_importance_shap.ipynb

```diff
@@ -367,7 +368,23 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "For guidelines on how to interpret such SHAP plots please see the [SHAP documentation](https://github.com/shap/shap).\n",
+    "In these SHAP summary plots, the color and orientation of the plotted values help understand\n",
+    "their impact on model predictions.\n",
+    "\n",
+    "Each dot in the plot represents a single instance of the given feature present in the data set.\n",
+    "The x-axis conveys the Shapley value, signifying the strength and directionality of the\n",
+    "feature's impact. The y-axis displays each feature in the model.\n",
+    "\n",
+    "The color coding implemented in these plots is straightforward: red implies a high feature value,\n",
+    "while blue denotes a low feature value. This color scheme assists in identifying whether\n",
+    "high or low values of a certain feature influence the model's output positively or negatively.\n",
+    "The categorical variables are grey colored.\n",
+    "\n",
+    "The Shapley value, exhibited on the horizontal axis, is oriented such that values on the\n",
```

Comment:
Should we move this comment up, to where the meaning of the x-axis is described?

Reply:
Sure! [8f92692](https://github.com/Quantco/metalearners/pull/21/commits/8f926921a03899faff28a764bd0effe01ac13484)